tests/migrations/test_commands.py

```diff
@@ -1861,6 +1861,58 @@ class Meta:
             )
             self.assertIn("Type 'exit' to exit this prompt", output)
 
+    def test_makemigrations_not_null_field_removable(self):
+        """
+        makemigrations messages when removing a NOT NULL field.
+        """
```

Comment:
```suggestion
```

Reply:
All the other tests have docstrings. ☹️

Reply:
I agree with David, this docstring is unnecessary.